views/dashboard.html

```diff
@@ -0,0 +1,40 @@
+{% extends "layout.html" %}
+
+{% block sidebar %}
+<li class="active"><a href="/">The Latest 100</a></li>
```

Comment:
This should be /dashboard, correct? Also this will need to change in the other html files as well. Might want to look into placing these in the layout.html file